walkthroughs/week-2-server/examples/server-stats/src/main/java/com/google/sps/servlets/ServerStatsServlet.java

```diff
@@ -26,7 +26,11 @@
 @WebServlet("/server-stats")
 public final class ServerStatsServlet extends HttpServlet {
 
-  private final Date startTime = new Date();
+    /**
```

Comment:
nit: looks like your editor is setting tabs to 4 spaces, but the file is set to  two space indentation for tabs. This is the classic tabs vs spaces dilemma. There's varying opinions on the right solution for this dilemma - as observed from a look at [stackoverflow](https://softwareengineering.stackexchange.com/questions/57/tabs-versus-spaces-what-is-the-proper-indentation-character-for-everything-in-e).

My personal preference is to use a language-specific code formatter that understands the syntax of the language and formats your code based on consistent criteria. for Java, there's https://github.com/google/google-java-format.